samples/containers/poetry/src/main/java/com/squareup/sample/poetry/views/MayBeLoadingScreen.kt

```diff
@@ -0,0 +1,20 @@
+package com.squareup.sample.poetry.views
+
+import com.squareup.sample.container.overviewdetail.OverviewDetailScreen
+import com.squareup.sample.container.panel.ScrimContainerScreen
+import com.squareup.workflow1.ui.WorkflowUiExperimentalApi
+import com.squareup.workflow1.ui.modal.HasModals
+
+@OptIn(WorkflowUiExperimentalApi::class)
+class MayBeLoadingScreen(
```

Comment:
nit: `MaybeLoadingScreen`

Reply:
Haha thats a great point - I wasn't even thinking of this in that descriptive sense but more of the prescriptive. This 'may' 'be loading'. English is funny :).